libsolidity/interface/FileReader.h

```diff
@@ -83,7 +84,51 @@ class FileReader
 		return [this](std::string const& _kind, std::string const& _path) { return readFile(_kind, _path); };
 	}
 
+	/// Normalizes a filesystem path in a way that removes small, inconsequential differences. Specifically:
```

Comment:
Maybe just sommarize in the header and put the full description in the cpp file?

Reply:
Done.